bika/lims/utils/analysisrequest.py

```diff
@@ -55,7 +55,7 @@ def create_analysisrequest(client, request, values, analyses=None,
         are read from the associated analysis service.
     """
     # Don't pollute the dict param passed in
-    values = deepcopy(values)
+    values = dict(values.items())
```

Comment:
What is the Acquisition wrapper that comes in in that case?
Not sure if this is the right place for this fix or if the issue should be tackled earlier, e.g. in `senaite.jsonapi`

Reply:
Any value of this dict could be passed as an object or aq_wrapper instead of a UID.  deepcopy does pickle/unpickle to enforce the full depth copy, and this is really not necessary; We just want to prevent modifying the top-level dict object that's passed in.

jsonapi can send UIDs if it wants to, though I would definitely prefer that this function is able to handle both UIDs, objects, and aq wrappers.